### DISCUSSION TITLE
"IncludePaths" vs packageRules->"matchPaths" ?

### DISCUSSION THREAD

**Discussion by em4o** [renovatebot/renovate#13966] [Request Help] [answered]
What is with a higher priority?
If I have a specific path inside "includePaths" and then I have a packageRules->"matchPaths" with a *?
Example:
includePaths: [test/myfolder]

packageRules {
      "matchPaths": [
        "test/**"
      ],
      "matchPaths": [
        "differentFolder/diff"
      ],
}

Question: For which paths this will be executed?
1. Only for "test/myfolder"
2. For all "test/**" and  "differentFolder/diff"
3.  For "test/myfolder" and  "differentFolder/diff"

**Comment by rarkins** [comment, marked as answer]
Only test/myfolder

If you restrict paths using includePaths then renovate won't attempt to extract dependencies from anywhere else. Therefore your matchPaths can never match something else

**Reply by em4o** [reply]
Thank you very much!
